### PR TITLE
Addresses 2028 NTR lung terms

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #2028 NTR lung terms

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31604,6 +31607,43 @@ AnnotationAssertion(rdfs:label obo:CL_4033043 "lung interstitial macrophage")
...
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033044 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033044 "2023-06-20T10:27:17Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31558434") Annotation(oboInOwl:hasDbXref "PMID:331922198") Annotation(oboInOwl:hasDbXref "PMID:36883528") rdfs:comment obo:CL_4033044 "In humans, deuterosomal cells have been noted to have enriched expression of the following: PLK4, FOXJ1, DEUP1, FOXN4, YPEL1, HES6 and CDC20B. Deuterosomal cells were termed after deuterosomes, the platforms of active centriole amplification.")
+AnnotationAssertion(rdfs:label obo:CL_4033044 "deuterosomal cell")
+SubClassOf(obo:CL_4033044 obo:CL_0002368)
+SubClassOf(obo:CL_4033044 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004802))
+
+# Class: obo:CL_4033045 (lung migratory dendritic cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34516781") obo:IAO_0000115 obo:CL_4033045 "A dendritic cell that captures antigens in the lungs and migrates to a lymph node or to the spleen to activate T cells.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31623]: "in the lungs" --> "in a lung"

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline suggestions for minor edits.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**